sql/mob_resistances.sql

```diff
@@ -4,10 +4,10 @@ DROP TABLE IF EXISTS `mob_resistances`;
 CREATE TABLE `mob_resistances` (
   `resist_id` smallint(4) unsigned NOT NULL,
   `name` tinytext, -- Purely for human readability
-  `slash_sdt` smallint(5) NOT NULL DEFAULT 1, -- sdt vs physical type
-  `pierce_sdt` smallint(5) NOT NULL DEFAULT 1,
-  `h2h_sdt` smallint(5) NOT NULL DEFAULT 1,
-  `impact_sdt` smallint(5) NOT NULL DEFAULT 1,
+  `slash_sdt` smallint(5) NOT NULL DEFAULT 0, -- sdt vs physical type
```

Comment:
I'm not going to ask...

Reply:
Some1 probably forgot to adjust the defaults